CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
 * Bug Fix: Identity property `externalId` was not properly rendering on `GET` and not handled consistently on `PUT` and `PATCH`
 * Enhancement: External JWT Signer Issuer & Audience Validation
 * Enhancement: Add ability to define local interface binding for link and controller dial
+* Enhancement: The mgmt API is now available via websocket. The stream commands are now available on `ziti fabric`
```

Comment:
Is this the entire management API or just a subset to support streaming?

Reply:
So, technically it's all of it, but we've already removed almost everything except streaming operations. I'll remove the last couple of non-streaming operations when I update the fabric tests to use the REST API instead. I'll probably also leave the inspect API since it's convenient to be able to run those asynchronously.